README.md

```diff
@@ -1 +1,73 @@
-# Hello-Microverse
\ No newline at end of file
+![](https://img.shields.io/badge/Microverse-blueviolet)
+
+# Project Name
+
+This is a first project from microverse
+
+## Built With
+
+- npm
+- vscode
+- html
+- css
+
+## Live Demo (if available)
+
+[Live Demo Link](https://livedemo.com)
+
+
+## Getting Started
+
+**This is an example of how you may give instructions on setting up your project locally.**
+**Modify this file to match your project, remove sections that don't apply. For example: delete the testing section if the currect project doesn't require testing.**
+
+
+To get a local copy up and running follow these simple example steps.
+
+### Prerequisites
+- Computer
+- Code editor
+
+### Setup
+- Clone the repository to your computer, or you download the zip file containing the repository files.
+- File editing can then be applied using any code editor.
+### Install
+- `npm install`
+
+### Usage
+
+
+### Deployment
+- To view the page right from the files cloned to your computer, open the index.html file from any browser.
+
+
+
+## Authors
+
+👤 **Author1**
+
+- GitHub: [@PedroH-Neves](https://github.com/PedroH-Neves)
+- Twitter: [@PedroHDN710](https://twitter.com/PedroHDN710)
+- LinkedIn: [Pedro Neves](https://www.linkedin.com/in/pedro-henrique-domingues-neves-8a5878244/)
+
+## 🤝 Contributing
+
+Contributions, issues, and feature requests are welcome!
+
+Feel free to check the [issues page](https://github.com/PedroH-Neves/Hello-Microverse/issues).
+
+## Show your support
+
+Give a ⭐️ if you like this project!
+
+## Acknowledgments
+
+- Hat tip to anyone whose code was used
+- Inspiration
+- etc
+
+## 📝 License
+
+This project is [MIT](./LICENSE) licensed.
```

Comment:
- [ ] The link to your license is broken cause your project doesn't have any license. Please add a license to your project and I recommend using the [MIT license](https://choosealicense.com/licenses/mit/), Set it up quickly by [using templates available on GitHub](https://docs.github.com/en/communities/setting-up-your-project-for-healthy-contributions/adding-a-license-to-a-repository). another resource is [any other license] :+1: (https://choosealicense.com/licenses/) . :smile: